data-structures-linked-lists/append.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-unused-vars */
+import LinkedList from './lib/linked-list.js';
+
+export default function append(list, value) {
+  const newNode = new LinkedList(value);
+  let lastNode = list;
+  if (lastNode) {
+    while (lastNode.next) {
+      lastNode = lastNode.next;
+    }
+  }
+  lastNode.next = newNode;
+  return lastNode;
```

Comment:
Generally, it's a good idea to match a documented function's behavior. That means if a function is supposed to return `undefined`, make sure you're not returning a defined value by accident:

---

![image](https://github.com/DylanPDao/c0323-code-solutions/assets/16143771/5c23542a-1099-4eee-8909-e7d79d9c2705)